gcloud/bigtable/client.py

```diff
@@ -149,6 +151,31 @@ def __init__(self, project=None, credentials=None,
         self._operations_stub_internal = None
         self._table_stub_internal = None
 
+    def copy(self):
+        """Make a copy of this client.
+
+        Copies the local data stored as simple types but does not copy the
+        current state of any open connections with the Cloud Bigtable API.
+
+        :rtype: :class:`.Client`
+        :returns: A copy of the current client.
+        """
+        data_stub = self._data_stub_internal
+        cluster_stub = self._cluster_stub_internal
+        operations_stub = self._operations_stub_internal
+        table_stub = self._table_stub_internal
+        try:
+            self._data_stub_internal = None
+            self._cluster_stub_internal = None
+            self._operations_stub_internal = None
+            self._table_stub_internal = None
+            return copy.deepcopy(self)
+        finally:
+            self._data_stub_internal = data_stub
+            self._cluster_stub_internal = cluster_stub
+            self._operations_stub_internal = operations_stub
+            self._table_stub_internal = table_stub
+
```

Comment:
I suppose. Past-tense me was likely trying to avoid hard-coding the constructor.

Though we'd want a copy of `self.credentials` (the rest are primitive types) to make this work.

Shall I make the change?